.gitignore

```diff
@@ -7,3 +7,4 @@ dist
 # misc
 .DS_Store
 npm-debug.log
+/.idea
```

Comment:
Good practice is to have a newline at end of file. You see, Git warns you about this. :) Could you please add a proper option for `.editorconfig` to ensure it? :)

Seems like this one:

```
insert_final_newline = true
```

Reply:
And then save file once again to make sure newline is inserted here and Git doesn't show a warning.

Reply:
Updated with new line and proper option in `.editorconfig`